File: app/src/main/java/stark/android/appbase/demo/MoreFragment.java
package stark.android.appbase.demo;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import stark.android.appbase.base.BaseFragment;

public class MoreFragment extends BaseFragment {

    public static final String TAG = "MoreFragment";

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_more, container, false);
    }
}

File: app/src/main/java/stark/android/appbase/demo/more/MoreFragment.kt
package stark.android.appbase.demo.more

import android.databinding.DataBindingUtil
import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

import stark.android.appbase.demo.R
import stark.android.appbase.demo.databinding.FragmentMoreBinding
import stark.android.appbase.fragment.BaseFragment

class MoreFragment : BaseFragment() {

    lateinit var mBinding: FragmentMoreBinding
    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        mBinding = DataBindingUtil.inflate(inflater, R.layout.fragment_more, container, false)
        return mBinding.root
    }

    companion object {

        val TAG = "MoreFragment"
    }
}
